src/modules/cmd_duration.rs

```diff
@@ -44,14 +44,15 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 }
 
 // Render the time into a nice human-readable string
-fn render_time(raw_seconds: u64) -> String {
-    // Calculate a simple breakdown into days/hours/minutes/seconds
+fn render_time(raw_millis: u128) -> String {
```

Comment:
Nitpick: "milli" doesn't bring to mind "millisecond" to me intuitively.
I'd be happy using "millisecond" or "ms" for a little more clarity. 😊 

```suggestion
fn render_time(raw_milliseconds: u128) -> String {
```

Reply:
I was using the same name that the Rust std library use. There is, for instance, the method `as_millis` in the `Duration` struct.
In any case, if you think that this is really misleading I will change it in the next commit.